Sources/HaCWebsiteLib/ViewModels/Hackathons/GameGig2017/GameGig2017.swift

```diff
@@ -0,0 +1,227 @@
+import HaCTML
+import Foundation
+
+// swiftlint:disable line_length
+
+
+extension String {
+  func idMangle() -> String {
+    return self.replacingOccurrences(of: " ", with: "-").lowercased()
+  }
+}
+
+// TODO: investigate if the information here can be extracted from the landing feature
+struct GameGig2017: Hackathon {
+  let gameEngines = [
+    ("Unreal Engine", "https://www.unrealengine.com"),
+    ("Unity", "https://unity3d.com/"),
+    ("LÖVE", "https://love2d.org/"),
+    ("GameMaker", "lhttps://www.yoyogames.com/gamemaker"),
+    ("raylib", "http://www.raylib.com/")
+  ]
+
+  let socialMediaLinks = [
+    ("Facebook Page", "https://www.facebook.com/hackersatcambridge"),
+    ("Facebook Event", "https://www.facebook.com/events/124219834921040/"),
+    ("Twitter", "https://twitter.com/hackersatcam")
+  ]
+
+  let tutorials = [
+    ("Web Dev with Mozilla", "https://globalgamejam.org/news/dev-web-mozilla")
+  ]
+
+  let GoboLogo = El.A[Attr.href => "http://studiogobo.com/",
+    Attr.target => "_blank"
+  ].containing(
+    El.Img[
+      Attr.src => Assets.publicPath("/images/sponsors/studiogobo-logo.svg"),
+      Attr.alt => "Studio Gobo"
+    ]
+  )
+
+  let ElectricSquareLogo = El.A[Attr.href => "https://www.electricsquare.com/",
+    Attr.target => "_blank"
+  ].containing(
+    El.Img[
+      Attr.src => Assets.publicPath("/images/sponsors/electricsquare-logo.svg"),
+      Attr.alt => "Electric Square"
+    ]
+  )
+
+  /**
+   * Creates a GameGigCard element, the title becomes its id (spaces are replaced with hyphons)
+   */
+  func GameGigCard(title: String, content: Nodeable) -> Node {
```

Comment:
For Cards? Like, literally cards. I used the same naming conventions as for the home page here.

Reply:
I just realised that reads more aggresively than I intended. What I'm saying is, how could I word it better? Because they best way to describe these is that they literally look like floating cards hovering over the background.